python3 eval_veri.py --datasets '/home/sunruina/face_recognition/data_set/ms_celeb_arcpaper_tfrecords/lfw.bin' --dataset_name 'lfw' --num_classes 85742 --ckpt_restore_dir '/home/sunruina/face_recognition/data_set/ms_celeb_arcpaper_tfrecords/face_real403_ckpt/Face_vox_iter_330000.ckpt'
